revel_test.go

```diff
@@ -0,0 +1,14 @@
+package revel
+
+import (
+	"net/http"
+)
+
+func NewTestController(w http.ResponseWriter, r *http.Request) *Controller{
```

Comment:
This is used by pretty much all of the test cases to create a controller. I thought this would be a better spot for this function